addon/serializer.js

```diff
@@ -0,0 +1,37 @@
+import Model from './orm/model';
+import Collection from './orm/collection';
+import { pluralize } from './utils/inflector';
+import extend from './utils/extend';
+
+class Serializer {
+
+  serialize(response, request) {
+    if (response instanceof Model) {
+      return this.serializeModel(response);
+
+    } else if (response instanceof Collection) {
+      return this.serializeCollection(response);
+
+    } else {
+      return response;
+    }
+  }
+
+  serializeModel(model) {
+    return this.constructor.root ? { [model.type]: model.attrs } : model.attrs;
```

Comment:
Excellent point - I was going to focus on getting AMS-like serializers working first, then hopefully use something like [jsonapify](https://github.com/alex94puchades/jsonapify) to convert the data to json api-like response.

I'll think a little more about this.